mayastor/src/bin/cli/rebuild_cli.rs

```diff
@@ -0,0 +1,256 @@
+//!
+//! methods to interact with the rebuild process
+
+use crate::context::Context;
+use ::rpc::mayastor as rpc;
+use clap::{App, AppSettings, Arg, ArgMatches, SubCommand};
+use tonic::Status;
+
+pub async fn handler(
+    ctx: Context,
+    matches: &ArgMatches<'_>,
+) -> Result<(), Status> {
+    match matches.subcommand() {
+        ("start", Some(args)) => start(ctx, &args).await,
+        ("stop", Some(args)) => stop(ctx, &args).await,
+        ("pause", Some(args)) => pause(ctx, &args).await,
+        ("resume", Some(args)) => resume(ctx, &args).await,
+        ("state", Some(args)) => state(ctx, &args).await,
+        ("progress", Some(args)) => progress(ctx, &args).await,
+        (cmd, _) => {
+            Err(Status::not_found(format!("command {} does not exist", cmd)))
+        }
+    }
+}
+
+pub fn subcommands<'a, 'b>() -> App<'a, 'b> {
+    let start = SubCommand::with_name("start")
+        .about("starts a rebuild")
+        .arg(
+            Arg::with_name("uuid")
+                .required(true)
+                .index(1)
+                .help("uuid of the nexus"),
+        )
+        .arg(
+            Arg::with_name("uri")
+                .required(true)
+                .index(2)
+                .help("uri of child to start rebuilding"),
```

Comment:
@tiagolobocastro is working on changes of bdev names etc but IMHO, we should not specify the URI rather the name of the child? (we can leave it for now just a general comment for a todo -- it goes for all the subcommands)

Reply:
Yes but not quite yet because this goes through the Nexus stuff and not the bdevs directly. `NexusChild` names are still URIs.

Reply:
Agreed :+1: 

I've added a todo for this